apps/moderation/serializers.py

```diff
@@ -51,8 +51,8 @@ def get_moderated_fields(self) -> list[dict]:
                     'name': field_name,
                     'type': field.__class__.__name__,
                     'editable': not getattr(field, 'read_only', False),
-                    'required': not getattr(field, 'required', False),
```

Comment:
Where did these two `not`s come from? 🤔

Reply:
No idea. Noticed them while discussing fields API behavior with @mattfredericksen.